### PR TITLE
[docs] add another kube-rbac-proxy example

### PR DESCRIPTION
**Pull request by stratnevpy**
## Description
Add another kube-rbac-proxy example

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Add another `kube-rbac-proxy` example.
impact_level: low
```

### CODE REVIEW

File: modules/300-prometheus/docs/FAQ.md [hunk 1]
```diff
@@ -132,6 +132,146 @@ spec:
 
 To enable secure access to metrics, we strongly recommend using **kube-rbac-proxy**.
 
+### An example of collecting metrics securely from an application inside a cluster
+
+To configure application metrics protection using `kube-rbac-proxy` and then build metrics from it using Prometheus tools, perform the following steps:
```
Comment (kirkonru) [inline modules/300-prometheus/docs/FAQ.md:137]:
```suggestion
Do the following to set up application metrics protection via the `kube-rbac-proxy` with the subsequent metrics scraping using Prometheus tools:
```

File: modules/300-prometheus/docs/FAQ.md [hunk 2]
```diff
@@ -132,6 +132,146 @@ spec:
...
+     name: d8:rbac-proxy
+   subjects:
+   - kind: ServiceAccount
+     name: rbac-proxy-test
+     namespace: default
+   ```
+
+   > The example uses the `d8:rbac-proxy` built-in Deckhouse `ClusterRole`.
+
+2. Create a configuration for `kube-rbac-proxy`:
```
Comment (kirkonru) [inline modules/300-prometheus/docs/FAQ.md:164]:
```suggestion
2. Create a configuration for the `kube-rbac-proxy`:
```

File: modules/300-prometheus/docs/FAQ.md [hunk 3]
```diff
@@ -132,6 +132,146 @@ spec:
...
+           namespace: default
+           apiVersion: v1
+           resource: services
+           subresource: proxy
+           name: rbac-proxy-test
+   ```
+
+   > Get more information on authorization attributes in the [Kubernetes documentation](https://kubernetes.io/docs/reference/access-authn-authz/authorization).
+
+3. Create `Service` and `Deployment` for your application, where `kube-rbac-proxy` takes the position of a sidecar container:
```
Comment (kirkonru) [inline modules/300-prometheus/docs/FAQ.md:186]:
```suggestion
3. Create `Service` and `Deployment` for your application with the `kube-rbac-proxy` as a sidecar container:
```

File: modules/300-prometheus/docs/FAQ.md [hunk 4]
```diff
@@ -132,6 +132,146 @@ spec:
...
+           image: quay.io/brancz/prometheus-example-app:v0.1.0
+           args:
+           - "--bind=127.0.0.1:8081"
+         volumes:
+         - name: config
+           configMap:
+             name: rbac-proxy-config-test
+   ```
+
+4. Assign the necessary rights to the resource to Prometheus:
```
Comment (kirkonru) [inline modules/300-prometheus/docs/FAQ.md:246]:
```suggestion
4. Add the necessary resource permissions to Prometheus:
```

File: modules/300-prometheus/docs/FAQ.md [hunk 5]
```diff
@@ -132,6 +132,146 @@ spec:
...
+     apiGroup: rbac.authorization.k8s.io
+     kind: ClusterRole
+     name: rbac-proxy-test-client
+   subjects:
+   - kind: ServiceAccount
+     name: prometheus
+     namespace: d8-monitoring
+   ```
+
+After step 4, your application's metrics should appear in Prometheus.
```
Comment (kirkonru) [inline modules/300-prometheus/docs/FAQ.md:273]:
```suggestion
After step 4, your application's metrics should become available in Prometheus.
```

**Review by kirkonru — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Failure: cherry-pick commit f5c2213b00ec88b9a6ee53d38b6929ff9348c64f to the branch [release-1.44](https://github.com/deckhouse/deckhouse/tree/release-1.44) failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/4605085714) for details.

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [4288](https://github.com/deckhouse/deckhouse/pull/4288) to the branch [release-1.44](https://github.com/deckhouse/deckhouse/tree/release-1.44) successful!

**Pull request merged**